src/ai/backend/storage/abc.py

```diff
@@ -25,6 +25,7 @@
 
 # Available capabilities of a volume implementation
 CAP_VFOLDER: Final = "vfolder"
+CAP_VFHOST_QUOTA: Final = "vhost"
```

Comment:
The value should be `"vfhost-quota"` because there may be more `vfhost-XXX` capability flags in the future.